packages/runtime/client-api/src/api/document.ts

```diff
@@ -256,15 +230,5 @@ export async function load(
     });
     const container = await loader.resolve({ url });
 
-    // The client-api CodeLoader doesn't actually read the proposed code details, so this doesn't really matter.
-    const codeDetails: IFluidCodeDetails = {
-        package: `@fluid-internal/client-api@${apiVersion}`,
-        config: {},
-    };
-
-    if (!container.existing) {
```

Comment:
I think this will break this code.
All other flows changed prior to that had distinct create & load flows (usually in the form of createNew boolean and appropriate IF). This code does not and utilizes single entry point - load() function to do both.
Which I assume means - we need to split entry point and provide createNew() entry-point and document it, including breaking.md mention.

Reply:
Will look closer at this. I was relying solely on the test results. We should sync offline about this, I have some questions about the scenario.